test/View/Helper/FlashMessengerTest.php

```diff
@@ -555,13 +565,51 @@ public function testRenderCurrentUsesCurrentAutoEscapeByDefault(): void
         $this->helper->addMessage('Foo<br />bar');
 
         $this->helper->setAutoEscape(false);
-        $displayAssertion = '<ul class="default"><li>Foo<br />bar</li></ul>';
+        $displayAssertion = '<ul class=""><li>Foo<br />bar</li></ul>';
         $display          = $this->helper->renderCurrent('default');
         $this->assertSame($displayAssertion, $display);
 
         $this->helper->setAutoEscape(true);
-        $displayAssertion = '<ul class="default"><li>Foo&lt;br /&gt;bar</li></ul>';
+        $displayAssertion = '<ul class=""><li>Foo&lt;br /&gt;bar</li></ul>';
         $display          = $this->helper->renderCurrent('default');
         $this->assertSame($displayAssertion, $display);
```

Comment:
These test changes are all to be reverted, as this highlights the BC break that was introduced